boards/arm/nrf5340_dk_nrf5340/nrf5340_dk_nrf5340_cpuapp.yaml

```diff
@@ -0,0 +1,14 @@
+identifier: nrf5340_dk_nrf5340_cpuapp
+name: NRF5340-DK-NRF5340-application-MCU
+type: mcu
+arch: arm
+toolchain:
+  - gnuarmemb
+  - xtools
+  - zephyr
+ram: 64
+flash: 256
+supported:
+  - i2c
+  - pwm
```

Comment:
Seems like a small list of supported features?

Reply:
For now yes, there will be more coming up, as we complete the support for this SoC